codalab/apps/health/models.py

```diff
@@ -6,3 +6,20 @@ class HealthSettings(models.Model):
     emails = models.TextField(null=True, blank=True)
     threshold = models.PositiveIntegerField(default=25, null=True, blank=True)
     congestion_threshold = models.PositiveIntegerField(default=100, null=True, blank=True)
+
+
+class StorageDataPoint(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    bucket_name = models.CharField(max_length=255)
+    # As reported by the actual bucket contents
```

Comment:
these comments could be `verbose_name`s? https://docs.djangoproject.com/en/3.1/ref/models/fields/#verbose-name